sklearn/feature_extraction/tests/test_text.py

```diff
@@ -35,6 +35,7 @@
 import pickle
 from io import StringIO
 
+from pytest import mark
```

Comment:
We can just `import pytest`,  and use `pytest.mark...` because latter in code from the name it's not clear that `mark`  object alone has anything to do with tests..